MagentoEcsFargate/magento_platform/magento_app_stack.py

```diff
@@ -0,0 +1,102 @@
+        self.magento_service = ecs_patterns.ApplicationLoadBalancedFargateService(self, "MagentoService",
+                                                                              cluster=_cluster,            # Required
+                                                                              cpu=2048,                    # Default is 256
+                                                                              desired_count=1,            # Default is 1
+                                                                              task_image_options=ecs_patterns.ApplicationLoadBalancedTaskImageOptions(
+                                                                                  image=ecs.ContainerImage.from_registry(
+                                                                                      "public.ecr.aws/bitnami/magento:2.4.4"),  # return RepositoryImage
+                                                                                  environment={
+                                                                                    "BITNAMI_DEBUG": "true",
+                                                                                    "MAGENTO_EXTRA_INSTALL_ARGS": "--enable-debug-logging=true",
+                                                                                    "MAGENTO_HOST": magento_lb.load_balancer_dns_name,
+                                                                                    "MAGENTO_DATABASE_HOST": database_instance.db_instance_endpoint_address,
+                                                                                    "MAGENTO_DATABASE_PORT_NUMBER": "3306",
+                                                                                    "MAGENTO_DATABASE_NAME": "MagentoDB",
+                                                                                    "MAGENTO_ELASTICSEARCH_HOST":es_domain.domain_endpoint,
+                                                                                    "MAGENTO_ELASTICSEARCH_PORT_NUMBER": "443",
+                                                                                    "MAGENTO_ELASTICSEARCH_USE_HTTPS": "yes",
+                                                                                    "MAGENTO_SEARCH_ENGINE": "elasticsearch7",
+                                                                                    "MAGENTO_DEPLOY_STATIC_CONTENT": "yes",
+                                                                                    "MAGENTO_EXTERNAL_HTTP_PORT_NUMBER": "80",
+                                                                                    "APACHE_HTTP_PORT_NUMBER" : "80"
+                                                                                    #"MAGENTO_SKIP_BOOTSTRAP":"yes", #Keep commented for the first run
+                                                                                    #"MAGENTO_SKIP_REINDEX" : "yes"  #Keep commented for the first run
```

Comment:
It is not very clear to me what the comments mean for these two lines. 

I think it's best to have a custom resource that trigger this only on the update. But that may be overkill for a blog post example. May be an additional doc in the README.md will be useful?  I could imagine having to switch this flag.

Reply:
doc add to the README.md. would like to have guidance on how to implement custom resource for a following commit

Reply:
I assume that Magento has an API or something to call to trigger bootstrap & reindex.

If that's the case, you can execute arbitrary code in Lambda function with custom resource. You can execute only on `on_create`  to trigger bootstrap & reindex. 

https://docs.aws.amazon.com/cdk/api/v1/docs/custom-resources-readme.html